src/models/User.js

```diff
@@ -1,57 +1 @@
-const mongoose = require("mongoose");
-const uniqueValidator = require("mongoose-unique-validator");
-const crypto = require("crypto");
-const secret = require("../config").secret;
-
-const UserSchema = new mongoose.Schema(
-    {
-        username: {
-            type: String,
-            lowercase: true,
-            unique: true,
-            required: [true, "can't be blank"],
-            match: [/^[a-zA-Z0-9]+$/, "is invalid"],
-            index: true
-        },
-        email: {
-            type: String,
-            lowercase: true,
-            unique: true,
-            required: [true, "can't be blank"],
-            match: [/\S+@\S+\.\S+/, "is invalid"],
-            index: true
-        },
-        bio: String,
-        image: String,
-        favorites: [{ type: mongoose.Schema.Types.ObjectId, ref: "Article" }],
-        following: [{ type: mongoose.Schema.Types.ObjectId, ref: "User" }],
-        hash: String,
-        salt: String
-    },
-    { timestamps: true }
-);
-
-UserSchema.plugin(uniqueValidator, { message: "is already taken." });
-
-UserSchema.methods.validPassword = function(password) {
-    const hash = crypto
-        .pbkdf2Sync(password, this.salt, 10000, 512, "sha512")
-        .toString("hex");
-    return this.hash === hash;
-};
-
-UserSchema.methods.setPassword = function(password) {
-    this.salt = crypto.randomBytes(16).toString("hex");
-    this.hash = crypto
-        .pbkdf2Sync(password, this.salt, 10000, 512, "sha512")
-        .toString("hex");
-};
-
-UserSchema.methods.toAuthJSON = function() {
-    return {
-        username: this.username,
-        email: this.email
-    };
-};
-
-mongoose.model("User", UserSchema);
+//use seqeulize schema
```

Comment:
No sequelize schema is implemented here. You might want to check that.

Reply:
yh, kingsley already did that, so i left tha blankt, once his PR is merged,  we'l make use of his own code